homeassistant/components/plugwise/binary_sensor.py

```diff
@@ -82,6 +84,7 @@ class PlugwiseBinarySensorEntityDescription(BinarySensorEntityDescription):
         name="Plugwise notification",
         icon="mdi:mailbox-up-outline",
         icon_off="mdi:mailbox-outline",
+        device_class=BinarySensorDeviceClass.PROBLEM,
```

Comment:
Notifications do not perse indicate a problem?

Reply:
in my case, they have always been at least noteworthy (and yes problems).
given the thumbs up on https://community.home-assistant.io/t/plugwise-core-and-custom-component/236250/292?u=mariusthvdb, I figured this might be dev team supported

Reply:
ok, thanks for reviewing.
Ill close